repo/content/committed_content_index.go

```diff
@@ -57,19 +57,15 @@ func (c *committedContentIndex) getContent(contentID ID) (Info, error) {
 	defer c.mu.Unlock()
 
 	info, err := c.merged.GetInfo(contentID)
-	if info != nil {
-		if c.shouldIgnore(info) {
-			return nil, ErrContentNotFound
-		}
-
-		return info, nil
+	if err != nil {
+		return nil, err
 	}
 
-	if err == nil {
+	if c.shouldIgnore(info) {
```

Comment:
Is `shouldIgnore(info)` false when `info == nil` ?

Reply:
at this point info won't be nil, that's due to the change in `merged.GetInfo()`